ipa-core/src/protocol/step.rs

```diff
@@ -8,6 +8,7 @@ pub enum ProtocolStep {
     #[step(child = crate::protocol::ipa_prf::step::IpaPrfStep)]
     IpaPrf,
     Multiply,
+    Add,
```

Comment:
Let's call this something else... like "send_random_data_to_helpers"

Reply:
branstorming out loud - maybe `AdditionInPrimeField` to indicate what it is actually doing and that it does not require communication?